doc-content/enterprise-only/getting-started-kogito/proc-kogito-migrate-drl-expose-ruleunits.adoc

```diff
@@ -0,0 +1,131 @@
+[id="proc-kogito-migrate-drl-expose-ruleunits_{context}"]
+= Exposing rule units with a REST endpoint using {QUARKUS}
+
+You can get started with migrating DRL project to {KOGITO} by using REST endpoint support and {QUARKUS}.
```

Comment:
I wouldn't mention Kogito here, since we are not using it yet in this step. Please rephrase this as

`You can get started exposing the former rules evaluation with a REST endpoint using {QUARKUS}.`

Reply:
Done